### DISCUSSION TITLE
Create README.es-ES.md for spanish documentation

### DISCUSSION THREAD

**Discussion by rpalaciosg** [Lugriz/pdfmake-wrapper#36] [Contributing]
Hi, I would like to know if you think its a good idea create a `README.es-ES.md` file for the spanish translation of the documentation?

I have a PR #35 in draft that you could review.

**Comment by Lugriz** [comment]
Hi @rpalaciosg! 

I think it's a good idea to add it. Thanks for you interest to contribute to this library.
